### DISCUSSION TITLE
invert mask?

### DISCUSSION THREAD

**Discussion by 4lt3r3go** [Bing-su/adetailer#368] [General]
i'm looking for a way to invert the action of aDetailer
i mean:
it should inpaint everywere EXCEPT the area that it found
Is it possible? i dont see this option

**Comment by Bing-su** [comment]
![chrome_nKk7x9zW15](https://github.com/Bing-su/adetailer/assets/37621276/21c37fc4-b82a-4f26-834a-0c5bf449f39e)
